src/main/java/org/gaul/s3proxy/Main.java

```diff
@@ -310,11 +310,12 @@ private static BlobStore createBlobStore(Properties properties,
         }
 
         BlobStoreContext context = builder.build(BlobStoreContext.class);
-        BlobStore blobStore = context.getBlobStore();
-        if (context instanceof RegionScopedBlobStoreContext &&
-                region != null) {
+        BlobStore blobStore;
+        if (region != null) {
```

Comment:
How does this differ from the previous code?  It seems unsafe since it no longer checks whether `context` is actually a `RegionScopedBlobStoreContext`.  Also I think OVH is Swift and thus should be a `RegionScopedBlobStoreContext`?